### PR TITLE
cleanup: fix errors in wrapped format and log capitalization in kubelet

### PR DESCRIPTION
**Pull request by wangyx1992**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
This PR fixes some errors in wrapped format and log capitalization in kubelet.

#### Which issue(s) this PR fixes:
Ref #15863

#### Does this PR introduce a user-facing change?
```release-note
NONE
```

### CODE REVIEW

**Comment by matthyx** [comment]
Hi @wangyx1992 thanks for the PR.
Could you check if this KEP on structured logging applies to your use case: https://github.com/kubernetes/enhancements/issues/1602

**Comment by wangyx1992** [comment]
> Hi @wangyx1992 thanks for the PR.
> Could you check if this KEP on structured logging applies to your use case: [kubernetes/enhancements#1602](https://github.com/kubernetes/enhancements/issues/1602)

Hi @matthyx, thanks for your review.  The KEP on structured logging aims at structuring logs from `klog `package, and this PR wants to format the errors produced by `fmt.Errorf()`, according to [https://github.com/kubernetes/kubernetes/issues/94695](url)

**Review by ehashman — COMMENTED** [review]
/lgtm
/triage accepted
/priority backlog

**Comment by wangyx1992** [comment]
This PR has been updated, then the `lgtm `label was removed. I'm looking for your review with pleasure. @matthyx @ehashman

**Comment by shubheksha** [comment]
/lgtm cancel

@wangyx1992 the beginning of the error message has to be capitalised as per the migration guide: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#remove-string-formatting-from-log-message

If we can revert those changes, it'll be good to go 👍

**Review by ehashman — COMMENTED** [review]
/lgtm

These are `fmt.Error`s, not structured log lines, hence this is correct.

**Review by mrunalp — APPROVED** [review]

**Comment by Random-Liu** [comment]
/lgtm
/approve

I learnt the new `%w` from this change. Thanks a lot!

File: cmd/kubelet/app/server.go [hunk 1]
```diff
@@ -500,11 +500,11 @@ func getReservedCPUs(machineInfo *cadvisorapi.MachineInfo, cpus string) (cpuset.
 
 	topo, err := topology.Discover(machineInfo)
 	if err != nil {
-		return emptyCPUSet, fmt.Errorf("Unable to discover CPU topology info: %s", err)
+		return emptyCPUSet, fmt.Errorf("unable to discover CPU topology info: %s", err)
```
Comment (Random-Liu) [inline cmd/kubelet/app/server.go:503]: Why not also update these to %w, and below?

**Pull request merged**